File: app/front/src/components/main/dto/kanrensha/getKanrenshaSeijidantaiCapsuleDto.ts
import { FrameworkCapsuleDto, type FrameworkCapsuleDtoInterface } from "seijishikin-jp-normalize_common-tool";
import { KanrenshaSeijidantaiMasterEntity, type KanrenshaSeijidantaiMasterEntityInterface } from "../../entity/kanrenshaSeijidantaiMasterEntity";

interface GetKanrenshaSeijidantaiCapsuleDtoInterface extends FrameworkCapsuleDtoInterface {

    /** 関連者政治団体マスタEntity */
    masterSeijidantaiEntity: KanrenshaSeijidantaiMasterEntityInterface;
}

class GetKanrenshaSeijidantaiCapsuleDto extends FrameworkCapsuleDto implements GetKanrenshaSeijidantaiCapsuleDtoInterface {

    /** 関連者政治団体マスタEntity */
    masterSeijidantaiEntity: KanrenshaSeijidantaiMasterEntityInterface;

    constructor() {
        super();

        this.masterSeijidantaiEntity = new KanrenshaSeijidantaiMasterEntity();
    }

}

export { type GetKanrenshaSeijidantaiCapsuleDtoInterface, GetKanrenshaSeijidantaiCapsuleDto }

File: app/back/src/main/java/net/seijishikin/jp/normalize/manage/kanrensha/dto/kanrensha/GetKanrenshaSeijidantaiCapsuleDto.java
package net.seijishikin.jp.normalize.manage.kanrensha.dto.kanrensha;

import java.io.Serializable;

import net.seijishikin.jp.normalize.common_tool.dto.FrameworkCapsuleDto;
import net.seijishikin.jp.normalize.manage.kanrensha.entity.KanrenshaSeijidantaiMasterEntity;

/**
 * 関連者政治団体格納Dto
 */
public class GetKanrenshaSeijidantaiCapsuleDto extends FrameworkCapsuleDto implements Serializable {

    /** Serialize id */
    private static final long serialVersionUID = 1L;

    /** 関連者政治団体マスタEntity */
    private KanrenshaSeijidantaiMasterEntity masterSeijidantaiEntity = new KanrenshaSeijidantaiMasterEntity();

    /**
     * 関連者政治団体マスタEntityを取得する
     * 
     * @return 関連者政治団体マスタEntity
     */
    public KanrenshaSeijidantaiMasterEntity getMasterSeijidantaiEntity() {
        return masterSeijidantaiEntity;
    }

    /**
     * 関連者政治団体マスタEntityを設定する
     * 
     * @param masterSeijidantaiEntity 関連者政治団体マスタEntity
     */
    public void setMasterSeijidantaiEntity(final KanrenshaSeijidantaiMasterEntity masterSeijidantaiEntity) {
        this.masterSeijidantaiEntity = masterSeijidantaiEntity;
    }

}
